content/en/docs/instrumentation/java/exporters.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Exporters
+weight: 50
+---
+
+In order to visualize and analyze your traces, you will need to export them to a
+backend such as [Jaeger](https://www.jaegertracing.io/) or
+[Zipkin](https://zipkin.io/). OpenTelemetry Java provides exporters for some
+common open source backends.
+
+If you use the Javaagent for
+[automatic instrumentation](/docs/instrumentation/java/automatic) you can learn
+how to setup exporters following the
+[Agent Configuration Guide](/docs/instrumentation/java/automatic/agent-config)
+
+For [manual instrumentation](/docs/instrumentation/java/manual), you will find
+some introductions below on how to set up backends and the matching exporters.
+
+## OTLP endpoint
+
+To send trace data to a OTLP endpoint (like the [collector](/docs/collector) or
+Jaeger) you'll want to use `opentelemetry-exporter-otlp`:
+
+<!-- prettier-ignore-start -->
+
+{{< tabpane text=true >}}
+{{% tab Gradle %}}
+
+```kotlin
+dependencies {
+    implementation 'io.opentelemetry:opentelemetry-exporter-otlp:{{% param javaVersion %}}'
+}
+```
+
+{{% /tab %}}
+{{% tab Maven %}}
+
+```xml
+<project>
+    <dependencies>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-exporter-otlp</artifactId>
+        </dependency>
+    </dependencies>
+</project>
+```
+
+{{< /tab >}}
+{{< /tabpane>}}
+
+Next, configure the exporter to point at an OTLP endpoint.
+
+If you use [SDK auto-configuration](/docs/instrumentation/java/manual/#automatic-configuration)
+all you need to do is update your environment variables:
+
+```shell
+env OTEL_TRACES_EXPORTER=otlp OTEL_METRICS_EXPORTER=otlp OTEL_LOGS_EXPORTER=otlp java -jar ./build/libs/java-simple.jar
+```
+
+In the case of [manual configuration] you can update the [example app](/docs/instrumentation/java/manual#example-app) like the
+following:
+
+<!-- prettier-ignore-end -->
+
+```java
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter;
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.exporter.otlp.logs.OtlpGrpcLogRecordExporter;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.opentelemetry.sdk.logs.export.BatchLogRecordProcessor;
+
+        SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder()
```

Comment:
Is the intent for this to be psuedocode or a blub that folks can copy and paste into their app? If a copy / pastable blurb, maybe consider wrapping this in a main function:
```
  public static void main(String[] args) {
    // Initialize opentelemetry early in application lifecycle
   SdkTracerProvider sdkTracerProvider = ...

   // ... continue with application startup
  }
```

Reply:
I copied the example app code from `manual.md` and put highlights for the changed lines